.eslintrc.js

```diff
@@ -65,8 +65,19 @@ module.exports = {
 		{
 			files: ['*.ts'],
 			parser: '@typescript-eslint/parser',
-			plugins: ['@typescript-eslint'],
-			extends: ['plugin:@typescript-eslint/recommended', 'prettier/@typescript-eslint', 'plugin:prettier/recommended'],
+			extends: [
+				'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+				'plugin:import/typescript',
+				'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+			],
+			parserOptions: {
+				ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
+				sourceType: 'module', // Allows for the use of imports
+			},
+			rules: {
+				'import/extensions': 'off',
```

Comment:
The default for this ist "never". Maybe "off" is not needed or even wrong?
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md